ttt_game.py

```diff
@@ -111,7 +111,7 @@ def vertical_three_of_a_kind(self, player):
             if self.board[0][i] == self.board[1][i] == self.board[2][i] == player:
                 return True
 
-    def diagonal_three_of_a_kind(self, player):
+    def is_any_diagonal_complete(self, player):
         """
```

Comment:
If you decide to change a function name always do so with accompanying test for that function.